### PR TITLE
remove no balance modals

### LINKED ISSUE #2043: Remove modals from delegation and undelegation
**Is your feature request related to a problem? Please describe.**

Currently clicking on those buttons we still show a modal warning the user that he has no stake or no stake in the validator. We should remove those and only use the actionmodal as agreed on.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #2043 

**Description:**

Thank you! 🚀

---

For contributor:

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: test/unit/specs/components/staking/__snapshots__/DelegationModal.spec.js.snap [hunk 1]
```diff
@@ -221,7 +222,7 @@ exports[`DelegationModal component matches snapshot has the expected html struct
               <span
                 class="tm-btn__value"
               >
-                Submit
+                Connecting...
```
Comment (jbibla) [inline test/unit/specs/components/staking/__snapshots__/DelegationModal.spec.js.snap]: is this correct?

Comment (fedekunze) [reply test/unit/specs/components/staking/__snapshots__/DelegationModal.spec.js.snap]: good catch

**Comment by jbibla** [comment]
💪 💪 💪 🤣 🤣 🤣 

screen shot 2019-03-05 at 1 44 41 pm

File: app/src/renderer/components/staking/UndelegationModal.vue [hunk 2]
```diff
@@ -36,10 +29,13 @@
...
       />
+      <p v-if="maximum > 0">
+        {{ denom }}s delegated: {{ atoms(maximum) }}
+      </p>
       <tm-form-msg
-        v-if="liquidAtoms === 0"
-        :msg="`doesn't have any ${denom}s`"
-        name="Wallet"
+        v-if="maximum === 0"
+        :msg="`don't have any delegation to this validator`"
```
Comment (jbibla) [inline app/src/renderer/components/staking/UndelegationModal.vue]: when would a user see this message?

Comment (fedekunze) [reply app/src/renderer/components/staking/UndelegationModal.vue]: when you try to undelegate and you don't have any delegations to that validator

File: app/src/renderer/components/staking/DelegationModal.vue [hunk 3]
```diff
@@ -168,7 +168,7 @@ export default {
       })
     },
     async submitForm(submitType, password) {
-      if (this.from === this.wallet.address) {
+      if (this.from === this.session.address) {
```
Comment (faboweb) [inline app/src/renderer/components/staking/DelegationModal.vue]: 👍

File: app/src/renderer/components/staking/UndelegationModal.vue [hunk 4]
```diff
@@ -29,10 +29,13 @@
...
```
Comment (jbibla) [inline app/src/renderer/components/staking/UndelegationModal.vue]:
```suggestion
        :msg="`don't have anything delegated to this validator`"
```

Comment (fedekunze) [reply app/src/renderer/components/staking/UndelegationModal.vue]: what about :msg="`don't have any ${bondDenom} delegated to this validator`"

Comment (jbibla) [reply app/src/renderer/components/staking/UndelegationModal.vue]: yeah that works too. the trouble is that `atom` should have an `s` and `stake` should not 😆

Comment (jbibla) [reply app/src/renderer/components/staking/UndelegationModal.vue]: #grammarisweird

**Pull request merged**